src/main/resources/com/ncc/aif/aida_ontology.shacl

```diff
@@ -655,75 +651,64 @@ aida:TextJustificationShape
     a sh:NodeShape ;
     sh:targetClass aida:TextJustification ;
 
-    sh:and (
-        aida:SharedJustificationShape
-        [
-            # exactly one start character offset within the document
-            sh:property [
-                sh:path aida:startOffset ;
-                sh:or ( [sh:datatype xsd:int] [sh:datatype xsd:integer] )  ;
-                sh:minCount 1;
-                sh:maxCount 1 ];
-
-            # and exactly one inclusive end offset within the document
-            sh:property [
-                sh:path aida:endOffsetInclusive ;
-                sh:or ( [sh:datatype xsd:int] [sh:datatype xsd:integer] )  ;
-                sh:minCount 1;
-                sh:maxCount 1 ] ;
-
-            # no other properties are allowed
-            sh:closed true ;
-
-            # except shared properties and rdf:type
-            sh:ignoredProperties (
-                aida:confidence
-                aida:dependsOnHypothesis
-                aida:privateData
-                aida:source
-                aida:sourceDocument
-                aida:system
-                rdf:type
-            )
-        ]
-    ) .
+    # exactly one start character offset within the document
```

Comment:
I don't understand what this change has to do with AIDA-678. Is it fixing some other issue?
Are we not using aida:SharedJustificationShape anymore?

Reply:
This is fixing a bug with SharedJustificationShape. It doesn't have anything to do with AIDA-678.